### ISSUE TITLE
Transient state variables list

### ISSUE THREAD

**Issue by khufkens** [bluegreen-labs/rsofun#1]
Create a list of transient state variables which should not be tracked in full (as a matrix), throughout spinup.

@lauramarques

**Comment by stineb** [comment]
In general: all flux variables
Specifically: All variables that are calculated of the form `y = f(x)`. Variables that must be kept track of for spinup are calculated of the form `y = f(x, y)`.

**Comment by khufkens** [comment]
Hi Beni,

Quick question, I'm tracking the flow of data through the model for a better understanding.

Now, at the end of `biosphere_lm3ppa.mod.f90` there is a call to `Zero_diagnostics( vegn  )` so all values set to 0 in this subroutine are of no importance to the next call to `biosphere_annual()` inversely all values in `vegn` not touched are of importance and should be retained (being cummulative in nature at some level i.e. y  = f(x,y) ).

Is this a correct reading of the routine? It would make sorting things easier.

**Comment by khufkens** [comment]
Hi Beni,

Quick question, I'm tracking the flow of data through the model for a better understanding.

Now, at the end of `biosphere_lm3ppa.mod.f90` there is a call to `Zero_diagnostics( vegn  )` so all values set to 0 in this subroutine are of no importance to the next call to `biosphere_annual()` inversely all values in `vegn` not touched are of importance and should be retained (being cummulative in nature at some level i.e. y  = f(x,y) ).

Is this a correct reading of the routine? It would make sorting things easier.

**Comment by stineb** [comment]
I think that's correct.

**Comment by khufkens** [comment]
Ok, perfect. This basically means that I need to export `vegn`, minus those positions. And I should be able to amend things using something after `initialize_vegn_tile( vegn, nCohorts)`, depending on a state switch other than or in addition to `myinterface%steering%init`. Hope to get this sorted this week.

**Comment by lauramarques** [comment]
Exactly, Koen. The ``Zero_diagnostics( vegn )`` is called at the beginning and also at the end of the ``biosphere_annual()`` subroutine.

**Comment by khufkens** [comment]
Thanks for confirming Laura, might circle back to you this week for some more pointers and ideas.
